ta/pkcs11/src/object.c

```diff
@@ -1060,6 +1060,11 @@ enum pkcs11_rc entry_set_attribute_value(struct pkcs11_client *client,
 	if (rc)
 		goto out;
 
+	/* Set key check value attribute */
+	rc = check_attr_checksum(head);
```

Comment:
I would prefer this to be called before `check_attrs_against_modification()`, for consistency:
first set all attributes, then check they comply, then create the object.